dev-lisp/gcl/gcl-2.6.10.ebuild

```
@@ -134,13 +133,10 @@ src_install() {
 		elisp-install ${PN} elisp/*.el
 	fi
 
-	insinto /usr/share/doc/${PF}
-	doins *.dvi
+	docompress -x /usr/share/doc/${PF}/*.dvi
 	if use athena; then
-		pushd xgcl-2 > /dev/null
-		insinto /usr/share/doc/${PF}
-		doins *.pdf
-		popd > /dev/null
+		dodoc xgcl-2/*.pdf
+		docompress -x /usr/share/doc/${PF}/*.pdf
```

```suggestion
		docompress -x "/usr/share/doc/${PF}/*.pdf"
```

Not sure if it works like that but the current solution is wrong, as it globs on top of system-installed files.

---

I am not sure, If I can follow. You mean accessing with ```docompress``` directly ```/usr/share/doc/${PF}/*.pdf```?

---

Should be now fixed, as it seems, that only 1-2 files are installed, so I can specify them directly. No globbing needed anymore.